.github/workflows/test.yml

```diff
@@ -92,3 +92,8 @@ jobs:
       if: matrix.platform == 'linux/amd64'
       run: |
         sudo -u testbot bash -lc 'cd rdk/services/slam/builtin && sudo go test -v -run TestOrbslamIntegration'
+
+    - name: Run viam-orb-slam3 integration tests
+      if: matrix.platform == 'linux/amd64'
+      run: |
+        sudo -u testbot bash -lc 'cd viam-orb-slam3 && sudo go test -v -run TestOrbslamIntegration'
```

Comment:
[opt] Should we consider running the integration tests using the `-race` flag?

Reply:
That is a great idea! I had forgotten about your troubles with race conditions when writing this. There is no possible race conditions that I see right now but there might be in the future.

Reply:
The same can apply to the test that's run above that. I will do the same in my code.

Reply:
The RDK one?

Reply:
Also, I just noticed that this test became flakey with the '--race' flag. It failed one run but passed the next. Will investigate.

Reply:
The DialMaxTimeoutSec is being set to 1 for testing.

Suggested fixes:
1) Change SetDialMaxTimeoutSecForTesting(1) to SetDialMaxTimeoutSecForTesting(2)
2) Ignore it. (it may have occurred because I updated this branch while the test was running..?)

Reply:
Increasing the max dial timeout sounds reasonable to me.